### PR TITLE
Refactor code quality issues

### PR DESCRIPTION
**Pull request by saif-deepsource**
Hey, I work at DeepSource, I ran DeepSource analysis on my fork of the repo and found [some improvements](https://deepsource.io/gh/saif-deepsource/apkleaks/) in the codebase, opening this PR so you can assess if our platform is right and helpful for you; happy to provide the tweaks separately otherwise. 

**Following are the changes-** 
- Removed length check in favour of truthiness of the object
- Removed unused imports
- Removed unnecessary return statement
- Use `sys.exit()` calls
- Add `.deepsource.toml` file for continuous analysis on bug risks/performance/code-quality issues on new changes.

Note: _I've fixed a few issues for now. You can find and fix other issues [here](https://deepsource.io/gh/saif-deepsource/apkleaks/)._

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```
Comment (dwisiswant0) [inline .deepsource.toml:8]: I think it's not necessary since we use LGTM for continuous review. :)

**Review by dwisiswant0 — CHANGES_REQUESTED** [review]
Hi, @saif-deepsource. It's lit! Please check review below & resolve conflicts first.

File: .deepsource.toml (hunk 1 above)
Comment (saif-deepsource) [reply .deepsource.toml]: Hey @dwisiswant0 thanks for your response.😀 I'm removing `.deepsource.toml` file if you don't want to try out DeepSource. 

However, I want to highlight a few points which explain how DeepSource wins over LGTM:
- DeepSource [Python analyzer](https://deepsource.io/directory/analyzer/python/) detects ~558 issues [set of rules] while LGTM covers only [148 rules](https://lgtm.com/projects/g/dwisiswant0/apkleaks/queries/?pack=com.lgtm%2Fpython-queries). 
- DeepSource offers an [Autofix feature ](https://deepsource.io/blog/autofix-future-of-code-reviews/)with which you can fix issues automatically in a couple of clicks. There are 88 Autofix available in Python analyzer. LGTM does not offer any sort of this feature.
- DeepSource offers [auto code formatter feature](https://deepsource.io/docs/transformer/), which helps auto transform your codebase to maintain code styling. Available for Python too. 🐍
- DeepSource guarantees less than 5% false positives.
- Much better integration and developer experience.

**Review by dwisiswant0 — APPROVED** [review]

**Comment by dwisiswant0** [comment]
Hey, thanks for your contributions. Since I didn't notice that.

Your DeepSource's also great things, may I use that on another project _(Go, maybe :P)_. \o/

**Comment by saif-deepsource** [comment]
Hey, @dwisiswant0 thanks for merging the PR. Yep, you can use our [Go analyzer](https://deepsource.io/docs/analyzer/go.html) for your Go projects. You can check other languages support [here](https://deepsource.io/docs/analyzer/).

**Pull request merged**